### ISSUE TITLE
"Jump forward" and "Jump back" navigation buttons are not removed from tab order when inactive

### ISSUE THREAD

**Issue by gautierchomel** [edrlab/thorium-reader#3110]
> The screen reader user believes these buttons should be functional because they can be accessed with the screen reader, even when visually disabled on the UI, and the user hasn't interacted with any in-text links that the buttons would apply to. This would violate WCAG 1.3.1 (Information and Relationships) and 2.1.1 (Keyboard).
> 
> Suggestion: putting these link-specific navigation buttons into a 'link navigation' region or similar would help clarify their purpose in context

reported by ebsco
